components/automate-cs-nginx/habitat/config/chef_https.conf

```diff
@@ -13,7 +13,7 @@
     ssl_verify_depth {{ cfg.ngx.http.ssl_verify_depth }};
 
     ssl_dhparam {{pkg.svc_data_path}}/dhparams.pem;
-    ssl_protocols {{cfg.ngx.http.ssl_protocols}};
+    ssl_protocols {{cfg.ngx.http.ssl_protocols}}; # nosemgrep
```

Comment:
Ha, ha, same comment/question here: since our code also uses an indirect reference for the version, and thus you have marked every occurrence with `nosemgrep`, would it not be better to just disable the `insecure-ssl-version` rule?